EIPS/eip-4345.md

```diff
@@ -4,7 +4,8 @@ title: Difficulty Bomb Delay to June 2022
 description: Delays the difficulty bomb to be noticeable in June 2022.
 author: Tim Beiko (@timbeiko), James Hancock (@MadeOfTin), Thomas Jay Rush (@tjayrush)
 discussions-to: https://ethereum-magicians.org/t/eip-4345-difficulty-bomb-delay-to-may-2022/7209
-status: Review
+status: Last Call
+last-call-deadline: 2021-12-08
```

Comment:
This should be at least 2 weeks out, but since this is going live on the 8th I'm going to let it slide this time.